src/authV2/azext_authV2/azext_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
what is the reason for this? This will override all the webapp extension commands to not have preview tag - & that is not correct. Please verify & remove.

Reply:
so this is a separate extension from the webapps extension and these commands are not preview, so I'm confused how it would override a diff extension commands?